kubeflow/core/jupyterhub.libsonnet

```diff
@@ -219,13 +219,30 @@ c.RemoteUserAuthenticator.header_name = 'x-goog-authenticated-user-email'",
           apiGroups: [
             "*",
           ],
-          // TODO(jlewi): This is very permissive so we may want to lock this down.
           resources: [
```

Comment:
Can you add a comment with a link to the file you have in the PR. This will help keep it in sync over time.